library/include/ck/library/tensor_operation_instance/gpu/grouped_convolution_forward.hpp

```diff
@@ -189,9 +192,17 @@ struct DeviceOperationInstanceFactory<ck::tensor_operation::device::DeviceGroupe
             {
                 add_device_grouped_conv2d_fwd_xdl_gnhwc_gkyxc_gnhwk_bf16_instances(op_ptrs);
             }
+#endif
+#ifdef CK_ENABLE_INT8
```

Comment:
Hi, I asked for remove gnhwc instances in previous PR.

Reply:
I sent you an msg earlier explaining why this was not deleted, but if there's no use case for this instance, I will delete it later. Will create another PR.

Reply:
deleted in latest commit